mux.go

```diff
@@ -112,9 +112,7 @@ func copyRouteConf(r routeConf) routeConf {
 	}
 
 	c.matchers = make([]matcher, 0, len(r.matchers))
-	for _, m := range r.matchers {
-		c.matchers = append(c.matchers, m)
-	}
+	c.matchers = append(c.matchers, r.matchers...)
```

Comment:
IMO copy is a bit more clear here:

```
c.matchers = make([]matcher, len(r.matchers))
copy(c.matchers, r.matchers)
```

Reply:
You're right, since we previously make sure there's enough space in the dst, we could just use copy instead. Will update the PR.